### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by LanaDX**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
-<!-- default file list -->
-*Files to look at*:
 
-* [CarsXtraScheduling.sql](./CS/WebSite/App_Data/CarsXtraScheduling.sql) (VB: [CarsXtraScheduling.sql](./VB/WebSite/App_Data/CarsXtraScheduling.sql))
-* [sql_data.sql](./CS/WebSite/App_Data/sql_data.sql) (VB: [sql_data.sql](./VB/WebSite/App_Data/sql_data.sql))
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# How to bind ASPxScheduler to MS SQL Server database
+# Scheduler for ASP.NET Web Forms - How to bind scheduler to MS SQL Server database
```
Comment (arminal) [inline Readme.md]:
```suggestion
# Scheduler for ASP.NET Web Forms - How to bind a scheduler to a MS SQL Server database
```

File: Readme.md [hunk 2]
```diff
@@ -1,25 +1,20 @@
...
+
+This example demonstrates how to bind the [ASPxScheduler](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxScheduler.ASPxScheduler) control to a database located at MS SQL Server 2005.
```
Comment (arminal) [inline Readme.md]: located at or located in?

File: Readme.md [hunk 3]
```diff
@@ -1,25 +1,20 @@
...
 
+## Files to Review
 
-<p>This example demonstrates how to bind the ASPxScheduler control to a database located at MS SQL Server 2005. The project includes the CarsXtraScheduling.sql file  which contains data table scripts. Sample data are attached in the sql_data.sql file, containing INSERT statements which  can be executed to generate the data.</p><p><strong>See Also:</strong><br />
-<a href="https://www.devexpress.com/Support/Center/p/E1450">E1450</a><br />
-<a href="https://www.devexpress.com/Support/Center/p/K18043">K18043</a><br />
-<a href="https://www.devexpress.com/Support/Center/p/E261">E261</a><br />
-<a href="https://www.devexpress.com/Support/Center/p/E409">E409</a></p>
+* [CarsXtraScheduling.sql](./CS/WebSite/App_Data/CarsXtraScheduling.sql) (VB: [CarsXtraScheduling.sql](./VB/WebSite/App_Data/CarsXtraScheduling.sql)) - contains data table scripts.
+* [sql_data.sql](./CS/WebSite/App_Data/sql_data.sql) (VB: [sql_data.sql](./VB/WebSite/App_Data/sql_data.sql)) - attaches sample data. The file contains `INSERT` statements that can be executed to generate the data.
```
Comment (arminal) [inline Readme.md]:
```suggestion
* [sql_data.sql](./CS/WebSite/App_Data/sql_data.sql) (VB: [sql_data.sql](./VB/WebSite/App_Data/sql_data.sql)) - attaches sample data. The file contains `INSERT` statements that can be executed to generate data.
```

File: Readme.md (hunk 2 above)
Comment (LanaDX) [reply Readme.md]:
```suggestion
This example demonstrates how to bind the [ASPxScheduler](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxScheduler.ASPxScheduler) control to a database located in MS SQL Server 2005.
```

**Pull request merged**